geoportailv3/static/js/layerfactoryservice.js

```diff
@@ -134,6 +135,7 @@ app.getWmsLayer_ = function(ngeoDecorateLayer) {
     var imageExt = app.getImageExtension_(imageType);
     var layer = new ol.layer.Image({
       source: new ol.source.ImageWMS({
+        crossOrigin: 'anonymous',
```

Comment:
In most cases (and in production) WMS GetMap requests will be sent to the WMS proxy in the application. I hope that adding `crossOrigin: 'anonymous'` won't cause any problem in that case.